src/Server.cpp

```diff
@@ -437,7 +439,7 @@ void cServer::ExecuteConsoleCommand(const AString & a_Cmd, cCommandOutputCallbac
 	{
 		return;
 	}
-
+	
```

Comment:
Seems that my IDE tabs all newlines to make them match the current indentation level... *Weeps*